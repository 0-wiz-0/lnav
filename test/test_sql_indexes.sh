#! /bin/bash

export YES_COLOR=1

run_cap_test ${lnav_test} -n \
    -c ";EXPLAIN QUERY PLAN SELECT * FROM syslog_log WHERE log_path GLOB '*/logfile_syslog.*'" \
    ${test_dir}/logfile_syslog.*

run_cap_test ${lnav_test} -n \
    -c ";SELECT * FROM syslog_log WHERE log_path GLOB '*/logfile_syslog.*'" \
    ${test_dir}/logfile_syslog.*

