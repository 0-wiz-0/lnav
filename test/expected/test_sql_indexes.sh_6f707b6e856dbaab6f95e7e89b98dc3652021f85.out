[1mlog_line [0m[1mlog_part [0m[1m       log_time         [0m[1m[7mlog_idle_msecs [0m[1mlog_level [0m[1mlog_mark [0m[1mlog_comment [0m[1mlog_tags [0m[1mlog_filters [0m[1mlog_msg_format [0m
[1m       0[0m[1m [0m[1m  <[0m[1mNULL[0m[1m>[0m[1m [0m[1m2009-07-20 22:59:26.000[0m[1m [0m[1m             0[0m[1m [0m[1minfo     [0m[1m [0m[1m       0[0m[1m [0m[1m     <[0m[1mNULL[0m[1m>[0m[1m [0m[1m  <[0m[1mNULL[0m[1m>[0m[1m [0m[1m     <[0m[1mNULL[0m[1m>[0m[1m [0m[1m               [0m
       2   <NULL> 2009-07-20 22:59:29.000              0 info             0      <NULL>   <NULL>      <NULL>                
